### PR TITLE
Create ASSETS.md with pre-commit scripts to maintain it

### PR DESCRIPTION
**Pull request by kubasobon**
Add ASSETS.md with status all Asset Inventory resources. [Take a look](https://github.com/elastic/cloudbeat/blob/d443dfc4a5fba8336a8672220630f2b343bf680a/internal/inventory/ASSETS.md).

A pre-commit action to update the table is included.

Closes https://github.com/elastic/security-team/issues/9387

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -70,6 +71,7 @@ require (
 	github.com/spf13/viper v1.19.0
 	github.com/stretchr/testify v1.9.0
 	github.com/tsg/go-daemon v0.0.0-20200207173439-e704b93fd89b
+	github.com/xuri/excelize/v2 v2.8.1
```
Comment (romulets) [inline go.mod:74]: why do we need `excelize` in the main go.mod if you created a separated go dependency context in `scripts/update_assets_md`?

File: tools/tools.go [hunk 2]
```diff
@@ -24,11 +24,14 @@ package tools
 
 import (
 	_ "github.com/elastic/go-licenser"
+	_ "github.com/ettle/strcase"
```
Comment (romulets) [inline tools/tools.go:27]: Why are imports in this file added?

File: go.mod (hunk 1 above)
Comment (romulets) [reply go.mod:74]: Same about `strcase`

**Review by romulets — COMMENTED** [review]
Very nice! I'm only not sure why the main go file has changed. But otherwise, lgtm!

File: internal/inventory/ASSETS.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+| Infrastructure | Security | Encryption | KMS Key | No ❌ |
+| Infrastructure | Security | Encryption | KMS Key Alias | No ❌ |
+| Infrastructure | Security | Secrets Management | Secrets Manager Secret | No ❌ |
+| Infrastructure | Security | Security Management | Security Hub | No ❌ |
+| Infrastructure | Storage | Disk | EBS Volume | No ❌ |
+| Infrastructure | Storage | Object Storage | S3 Bucket | No ❌ |
+| Infrastructure | Storage | Snapshot | EBS Snapshot | No ❌ |
+| Infrastructure | Storage | Snapshot | RDS Snapshot | No ❌ |
+
+Progress: 12% (10/83)
```
Comment (oren-zohar) [inline internal/inventory/ASSETS.md:89]: Can the progress be on top? Can we break it down per resource type?

File: go.mod (hunk 1 above)
Comment (kubasobon) [reply go.mod]: Moved dependencies to a separate `go.mod/sum` 👍

File: tools/tools.go (hunk 2 above)
Comment (kubasobon) [reply tools/tools.go]: Moved dependencies to a separate `go.mod/sum` 👍

File: internal/inventory/ASSETS.md (hunk 3 above)
Comment (kubasobon) [reply internal/inventory/ASSETS.md]: Added, broken down by category.

**Comment by kubasobon** [comment]
Remarks addressed. [Fresh preview](https://github.com/elastic/cloudbeat/blob/d443dfc4a5fba8336a8672220630f2b343bf680a/internal/inventory/ASSETS.md)

**Review by romulets — APPROVED** [review]

**Pull request merged**